infrastructure/terraform/modules/alb.tf

```diff
@@ -49,7 +49,7 @@ resource "aws_lb_target_group" "conversion-tg" {
     git-origin      = var.git-origin
   }
 }
-
+*/
```

Comment:
Would it be better to just remove this if it's commented out?

Reply:
Since this is how it's configured in DevPre & Prod we'll leave this is and will remove it all once DevPre & Prod only support https.